clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/settings/views/camera/CameraSettingsViewMediatorBase.as

```diff
@@ -0,0 +1,10 @@
+package org.bigbluebutton.lib.settings.views.camera {
+	
+	import robotlegs.bender.bundles.mvcs.Mediator;
+	
+	public class CameraSettingsViewMediatorBase extends Mediator {
+		
+		[Inject]
+		public var view:CameraSettingsViewBase;

```

Comment:
Common parts moved from `CameraSettingsViewMediatorAIR` to `CameraSettingsViewMediatorBase`